Perform data cleaning

DunngenMaster opened the pull request:
Performing data cleaning including null values and check proportionality

DunngenMaster committed:
Perform data cleaning

Performing data cleaning including null values and check proportionality

bringesh2001 commented:
Hey Performed data analysis at initial level. Replace null with mean in case any suggestion guys

bringesh2001 committed:
Initiate visualization

DunngenMaster committed:
Delete initial-model.ipynb

nandavihari commented:
Which predictors contain null values and is there any situation that why they are having null values?

bringesh2001 committed:
1.Checked duplicate values.
2.Removed unnecessary ,duplicated columns like id,timestamp,...
3. Box plot for predictor variables for outlier detection.

bringesh2001 committed:
1.some insights from plots

bringesh2001 committed:
1.changed the bins of price from 1 to 100 dollars , to include every price
2.checked relation b/w temperature and price
3.checked relation b/w windspeed and price
